test/visual/number-field.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
```

Comment:
Issues like this should be fixed in skeleton and then applied w/o CR to all components we have: https://github.com/vaadin/vaadin-element-skeleton/blob/master/demo/advanced.html